src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -148,26 +163,42 @@ class TaskForm extends React.Component<Props, State> {
     };
 
     handleValidSubmit = (): void => {
-        const { id, createTask, editTask, editMode, taskType } = this.props;
-        const { message, approvers, dueDate, isValid } = this.state;
+        const { id, createTask, approvers, editTask, editMode, taskType } = this.props;
+        const { message, approvers: currentApprovers, dueDate, isValid } = this.state;
         const dueDateString = dueDate && dueDate.toISOString();
 
         if (!isValid) return;
 
         this.setState({ isLoading: true });
 
         if (editMode === TASK_EDIT_MODE_EDIT && editTask) {
+            // Added assignees are the ones in state that weren't in the prop
+            // Assignees to remove are the ones in the prop that cannot be found in state
+            const approverIds = approvers.map(approver => approver.id);
+            const currentApproverIds = currentApprovers.map(currentApprover => currentApprover.id);
+
             editTask(
                 {
                     id,
                     description: message,
                     due_at: dueDateString,
+                    addedAssignees: convertAssigneesToSelectorItems(
+                        currentApprovers.filter(currentApprover => approverIds.indexOf(currentApprover.id) === -1),
```

Comment:
We can probably make good use of [helper utils](https://lodash.com/docs/4.17.11#intersection) for situations like this one.

Reply:
good call. will add to a post-launch tracker (once we verify we have the desired behavior implemented)